database/migrations/2021_11_22_093555_migrate_thread_feed_to_timestamp.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use App\Models\Thread;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class MigrateThreadFeedToTimestamp extends Migration
+{
+    public function up()
+    {
+        Schema::table('threads', function (Blueprint $table) {
+            $table->timestamp('last_active_at')
+                ->after('slug')
+                ->nullable()
+                ->index();
+        });
+
+        // Iterate over all the threads and update the `last_active_at` timestamp to
+        // the latest reply `created_at` or thread `created_at` if no reply exists.
+        Thread::with(['repliesRelation' => function ($query) {
+            $query->latest();
+        }])
+            ->lazy()
+            ->each(function ($thread) {
+                $thread->update([
+                    'last_active_at' => ($reply = $thread->replies()->first()) ? $reply->created_at : $thread->created_at,
+                ]);
+            });
```

Comment:
Think we need to wrap this in the unit testing check. This doesn't needs to be run for unit tests, just one time in production.